docs/components/modeler/dmn/dmn.md

```diff
@@ -0,0 +1,61 @@
+---
+id: camunda-modeler-dmn
+title: Editing DMN in Camunda Modeler
+---
+
+# Quickstart
+
+## Create new DMN Decision Requirement Diagram
+
+![New DMN Diagram](./img/create-dmn.png)
+
+To start modeling, create a new DMN 1.3 diagram for the Camunda Platform by selecting **Create diagram > Create new DMN diagram (Camunda Platform)** in the top-level menu.
+
```

Comment:
This is not Cloud first, similar to BPMN. Can you modify the diagram and sentence to be process engine agnostic?

Reply:
Should this be labeled Platform-only? Should we have DMN content within this PR per the docs vision, or should this component be removed for now?

Reply:
Since we are moving all Modeler content over, please keep it and label it Platform-only. My mistake, my brain was moving too fast and forgot we don't have DMN for Cloud quite yet!